src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -3543,6 +3546,11 @@ type QueryType {
         """
         filterByModes: [Mode]
 
+        """
+        Only include places that match one of the given network names.
+        """
+        filterByNetworkNames: [String]
```

Comment:
I think we have so far referred to this just as network. Even though it's slightly confusing, maybe we should stick to that naming convention (also rename in other places in this pr). You also need to specify in the documentation above this that this refers to rental networks as this query is used for other purposes as well.

Reply:
Renamed to filterByNetwork and added rental network to the description.